homeassistant/components/energy/sensor.py

```diff
@@ -220,6 +222,7 @@ def __init__(
         self._config = config
         self._last_energy_sensor_state: State | None = None
         self._cur_value = 0.0
+        self._ready = asyncio.Event()
```

Comment:
This is maybe not the best name, any suggestion?
Also, should this be a generic feature of `Entity`?

Reply:
Don't make it generic because we have never needed it before